vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,199 @@
+
+### Entando Component Repository (ECR)
+
+* Added support for bundle upgrade, downgrade and update.
+* Bundle improvements; in particular, bundle plugin security and reliability via name signing.
+* It's now possible to specify standard and secret [environment variables](../../tutorials/devops/plugin-environment-variables.md) for a bundle plugin (microservice).
+* Now offers experimental support for [bundle plugin configuration profiles](../../tutorials/devops/plugin-configuration.md).
+* Now offers better support for bundles and bundle plugins with very long names. 
+* ENTANDO_PLUGIN_NAME has been renamed to ENTANDO_RESOURCE_NAME.
+* Implemented better management of certain error conditions during installation and updates.
+* Introduced system-level bundles, which allow the installation of global static resources.
+* The ECR client now explicitly marks bundles that are installed with an amended installation plan and allows the plan to be recalled.
+* Added support for Oracle 18c.
+* Fixed a bug that caused the ECR to indefinitely stay in busy-state due to particular failure conditions.
+* Fixed a bug that (under certain conditions) prevented the bundle plugin pod from properly restarting when scaled down, then back up again.
+* The ECR now correctly deletes the plugin pod when the bundle is uninstalled, but the deployment and volumes are retained.
+* The EntandoPlugin controller will now rollback if there is a timeout.
+* A failing pod in an OpenShift installation will trigger logging to capture error locations. The microservice deployment will automatically scale to 0 and disappear.
+
+### ent (entando-cli)
+
+* Reworked the `ent diag` command to also export CRs, ingresses, deployments, pod "previous" logs, namespace events, etc.
+* ent is now able to sync the kubectl version of the current profile with the server version (`ent auto-align-kubectl`).
+* Added helper to fetch logs from bundle plugins (`ent prj be-log`).
+* In order to run node-based Entando tools, ent now installs and uses a private copy of node at user-level.
+* A jq internal dependency is now installed at user-level, privately to ENT.
+* Added the command `ent pkg` to manage packages installed at user-level, with initial support for jq and K9s.
+* Added the helpers `get bundle id` and `get plugin id` to `ent ecr` and `ent prj` to determine the names of the resources related to a bundle.
+* Added the command `ent gen-secret`  to `ent ecr` and `ent prj` to help generate Secrets for the plugins.
+* Added the `ent ecr cr` and `ent ecr deploy` subcommands.
+* ent now installs the entando-releases dependencies at release level, allowing multiple profiles to share them.
+* Fixed a bug that caused ent to git-ignore the "bundle" subdirs in the MFE branches.
+* `ent pbs-init` now accepts SSH/git URLs.
+* `ent check-env` now considers Docker and Java mandatory dependencies (still skippable with "--lenient").
+
+### ent Quickstart
+
+* Updated to support release 7.0.0.
```

Comment:
Let's remove this section, drop the first bullet, and move the 2nd bullet to the previous section on ent.